src/cargo.js

```diff
@@ -72,12 +84,10 @@ export const handleCargo = async function (self, result) {
 
   const depFile = wasmFile.slice(0, -'.wasm'.length) + '.d';
   const depContents = await fse.readFile(depFile, 'utf-8');
-  for (let line of depContents.split('\n')) {
-    if (line.startsWith(wasmFile) || (jsFile && line.startsWith(jsFile))) {
-      for (let dep of line.split(/:\s+/)[1].split(/\s+/)) {
-        self.addDependency(dep);
-      }
-    }
+  const deps = parseDependencies(depContents, wasmFile.replace(/ /g, '\\ '));
```

Comment:
Can the `wasmFile.replace(/ /g, '\\ ')` bit be done inside of `parseDependencies`?  To me it seems like an implementation detail of the dependency file format.

Reply:
Sure thing. That definitely makes more sense.